src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 120;
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user = new User();
+        user.setLogin("kitty95");
+        user.setAge(MIN_AGE);
+        user.setPassword("123456");
+    }
+
+    @Test
+    void register_addValidUser_Ok() {
+        int sizeBefore = Storage.people.size();
+        registrationService.register(user);
+        int sizeAfter = Storage.people.size();
+        assertTrue(Storage.people.contains(user));
+        assertEquals(1, sizeAfter - sizeBefore);
+    }
+
+    @Test
+    void register_addTwoValidUsersWithSameLogin_Ok() {
+        User secondUser = new User();
+        secondUser.setLogin(user.getLogin());
+        secondUser.setAge(MIN_AGE + 15);
+        secondUser.setPassword("00123456");
+        registrationService.register(user);
+        assertThrows(RegistrationException.class, () -> registrationService.register(secondUser));
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(RegistrationException.class, () -> registrationService.register(null));
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        user.setLogin(null);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_invalidLogin_notOk() {
```

Comment:
why this login is invalid? add explanation in the method name, e.g. `shortLogin`, `loginLessThanMinLength` etc.
fix all similar places in the code please.